game/templates/game/board.html

```
@@ -177,7 +177,10 @@
                     (PvP)</button>
                 <button class="btn btn-gold" id="welcomeAIBtn" style="padding: 12px; font-size: 1.05rem;">Play vs
                     AI</button>
-
+                <button class="btn btn-gold" id="welcomeDailyPuzzleBtn"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)